internal/application/filesystem.go

```diff
@@ -53,14 +53,14 @@ const (
 	m2kQAServerMetadataFile                 = "." + appNameShort + "qa"
 	m2kPlanOngoingFile                      = "." + appNameShort + "plan"
 	apiServerPort                           = 8080
-	timestampRegex                          = `time="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z"`
 	loglevelRegex                           = `level=([a-z]+) `
 	newfilesMetadataFileName                = "newfiles.txt"
 	defaultDirectoryPermissions os.FileMode = 0777
 	defaultFilePermissions      os.FileMode = 0777
 )
 
 var (
+	timestampRegex   = regexp.MustCompile(`time="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z"`)
 	validArchiveExts = []string{".zip", ".tar", ".tgz", ".gz"}
```

Comment:
Will this support .tar.gz and .tar.xz? I assume at least .tar.gz is quite common.

Reply:
I didn't change the regex. I just moved the regex as is to a `var` block so it can be reused.

Reply:
That regex is not dealing with any archive file extensions.
It's being used to replace the timestamp in log lines with the app name.

Reply:
Okay, we can extend the archive formats we support some other time then.